internal/shard/shard_thread.go

```diff
@@ -7,12 +7,15 @@ import (
 	"sync"
 	"time"
 
+	eval2 "github.com/dicedb/dice/internal/eval"
```

Comment:
nit: Tried moving it locally to `"github.com/dicedb/dice/internal/eval"` server runs fine. Please let me know if I'm missing something.

Reply:
no nothing, caused by some refactoring blues of moving stuff around...
thanks for catching